README.md

```diff
@@ -1,73 +1,68 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Set up a "Hello world" project with two apps
 
-> Description the project.
+> This is a project divided in two apps.
 
+The first app is a Rails back-end that provides an API that shows a random greeting.
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
+The second app is a React front-end that gets the greeting from the Rails API and print the text.
 
-## Live Demo (if available)
+Here is the [link](https://github.com/LOctavio/hello-react-front-end/pull/1) to the react APP
 
-[Live Demo Link](https://livedemo.com)
+## Built With
 
+- Ruby on rails
+- React
+- Redux
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
+#### Prerequisites
+
+- Ruby >= 3.0.1 
+- Ruby on Rails >= 7.0.1
 
 ### Setup
 
-### Install
+- Open the terminal
+- Open the directory where you want to store the project
+- Clone the back-end project runing 'git clone https://github.com/LOctavio/hello-rails-back-end' on the terminal
+- Clone the  front-end project runing 'git clone https://github.com/LOctavio/hello-react-front-end' on the terminal
 
-### Usage
+### Install
 
-### Run tests
+- Open the project directory on the terminal
+- Inside the project's directory run 'bundle install' to install all the dependencies
+- run 'rails db:migrate' to migrate the database
 
-### Deployment
+### Usage
 
+- Open the project directory on the terminal
```

Comment:
- [ ] I think some steps are missing in this README file like how we can create a database and how to run the seed file. Please make sure to include these steps so that everyone can run your project easily.